lc-hax/Scripts/Modules/TriggerMod.cs

```diff
@@ -83,7 +83,7 @@ sealed class TriggerMod : MonoBehaviour, IEnemyPrompter {
                 break;
             }
 
-            if (collider.GetComponentInParent<EnemyAI>().Unfake() is EnemyAI enemy && Setting.EnablePhantom) {
+            if (collider.GetComponentInParent<EnemyAI>().Unfake() is EnemyAI enemy && Setting.EnablePhantom && PossessionMod.Instance?.PossessedEnemy != enemy) {
```

Comment:
What's this?

Reply:
As they said, sometimes the enemy you possessed would be in the way to trigger what you wanted, so I added a quick flag to check if the enemy is possessed and if it is it won't be able to be triggered. I did the same thing for ESP